exercises/ex_atm.py

```diff
@@ -0,0 +1,137 @@
+# ATM Exercise
+
+import sys
+
+FILE_PATH = 1
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+
+
+def read_file_move_to_dict(file_path):
+    """Read file and move atm information to dictionary.
+
```

Comment:
the actions of this function look a bit complicated, and it's hard to understand what you are doing.
Add a description here of what you expect the file to look like.